src/reducers/reducer_fetch_careers.js

```diff
@@ -0,0 +1,40 @@
+import {
+  FETCH_CAREERS_PENDING,
+  FETCH_CAREERS_FULFILLED,
+  FETCH_CAREERS_REJECTED,
+} from '../constants';
+
+const initialState = {
+  data: [],
+  dataFetched: false,
+  isFetching: false,
+  error: false,
+};
+
+const fetchCareersReducer = (state = initialState, action) => {
+  switch (action.type) {
+    case FETCH_CAREERS_PENDING:
+      return {
+        ...state,
+        data: [],
+        isFetching: true,
+      };
+    case FETCH_CAREERS_FULFILLED:
+      return {
+        ...state,
+        data: action.payload,
```

Comment:
Keep the state as concise and readable as possible - swap out `data` and don't just place the whole response object in state. 
`careerCards: action.payload.data,`